Runtime/Scripts/SelectionGroup.cs

```diff
@@ -13,6 +13,21 @@ namespace Unity.SelectionGroups
     /// <summary>
     /// A component to group a number of GameObjects under a common name.
     /// </summary>
+    /// <remarks>
+    /// The SelectionGroup component manages a list of GameObjects that are part of the group. It is created and managed using the Selection Group window.
```

Comment:
```suggestion
    /// The SelectionGroup component manages a list of GameObjects that are part of the group. It's created and managed using the **Selection Groups** window.
```
The manual part of the docs seems to call it the Selection Groups window, plural? [Style convention](https://docs-style-guide.unity.com/format/text-formatting#bold-text) is to have window names in bold also.